packages/grafana-ui/src/components/Table/Table.story.tsx

```diff
@@ -93,6 +94,25 @@ function buildData(theme: GrafanaTheme2, config: Record<string, FieldConfig>): D
   return prepDataForStorybook([data], theme)[0];
 }
 
+function buildFooterData(data: DataFrame): ReactNode[] {
+  const values = data.fields[3].values.toArray();
+  const valueSum = values.reduce((prev, curr) => {
+    return prev + curr;
+  }, 0);
+
+  const valueField = data.fields[3];
+  const displayValue = valueField.display ? valueField.display(valueSum) : valueSum;
+  const val = valueField.display ? formattedValueToString(displayValue) : displayValue;
+
+  const valCell = (
+    <ul style={{ listStyle: 'none' }}>
+      <li>sum: {val}</li>
+      <li>min: {5.2}</li>
+    </ul>
+  );
+  return ['Totals', <div key={1}>10</div>, null, valCell, '100%'];
```

Comment:
Wounder if we can type this or have some more structure around this rather than sending in React.Node[] 

Maybe  a React.FooterItem  that can have a string value, or a key value arrray

Reply:
If Table took in a Table.FooterItem[]  then Table could also calculate footerHeight based on the max item with the most key-value pairs